### DISCUSSION TITLE
projects-discord-webhooks

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#6] [Fan Comments]
# projects-discord-webhooks

{{ discord_webhooks }}

https://mrpowergamerbr.com/br/projects/discord-webhooks/

**Comment by Sweet12345678** [comment]
Términos de las DONACIONES 

Una donación es totalmente voluntaria, se puede jugar sin donar y en ningún caso se devolverá el dinero.

Nosotros te entregaremos lo que hayas elegido a la hora de donar, pero siempre recalcamos que no es nuestra obligación y en caso de ser baneado permanentemente del servidor se te borrarán todos los bienes.

DEFINICIÓN DE DONACIÓN:
Cosa que se da a una persona de forma voluntaria y sin esperar premio ni recompensa alguna, especialmente cuando se trata de algo de valor.


Al donar recibirás el rol de @donador , en caso de aportar una gran cantidad económica al desarrollo del servidor recibirás @donador vip

**Comment by Yuriiiiiiksks** [comment]
Miau

**Reply by Yuriiiiiiksks** [reply]
Uijai

**Comment by Yuriiiiiiksks** [comment]
>:(((

**Comment by bibifofa** [comment]
kkkk

**Comment by TiagoMoscoso** [comment]
gigachad

**Comment by TiagoMoscoso** [comment]
gigachad

**Comment by cardianofelipe** [comment]
Inicio

**Comment by S0ph14v1ck** [comment]
hehe
